enhancements/two-nodes-openshift/2no.md

```diff
@@ -300,26 +369,29 @@ This proposal is an alternative architecture to Single-node and MicroShift, so i
 
 1. Risk: If etcd were to be made active on both peers during a network split, divergent datasets would be created
    1. Mitigation: RHEL-HA requires fencing of a presumed dead peer before restarting etcd as a cluster-of-one
-   1. Mitigation: Peers remain inert (unable to fence peers, or start cri-o, kubelet, or etcd) after rebooting until they can contact their peer
+   2. Mitigation: Peers remain inert (unable to fence peers, or start cri-o, kubelet, or etcd) after rebooting until they can contact their peer
 
-1. Risk: Multiple entities (RHEL-HA, CEO) attempting to manage etcd membership would cause an internal split-brain
+2. Risk: Multiple entities (RHEL-HA, CEO) attempting to manage etcd membership would cause an internal split-brain
```

Comment:
it would not create a split-brain, it would just simply create multiple (similar) containers that would try to listen on the same port and run on the same database files